active/CommandBar/new-command-bar-features.md

```diff
@@ -0,0 +1,269 @@
+
+All child items of a RadioMenuFlyoutSubItem should be of type [RadioMenuFlyoutItem](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.radiomenuflyoutitem?view=winui-2.5) (although this is not enforced by the API). If a child item is selected, the parent item will show as selected. All child items should have the same GroupName as siblings of the parents. 
+
+The parent (RadioMenuFlyoutSubItem) cannot be selected. Thus it does not have GroupName or IsChecked properties. It can only achieve selection (and show the selection indicator) if it is bubbled up from one of its child items. Therefore, only one item out of the child items and other top-level RadioMenuFlyout items in the same Group can be selected at a time. 
+
+If a child item becomes selected, the selection indicator will bubble up to its parent when its MenuFlyout is closed. If a child is selected and the MenuFlyout is open, both the parent and child will show the selection indicator (similar to a hierarchical NavigationView).
+
+These are the three main visual states for this new API, using the same Xaml Controls Gallery sample that was previously shown:
+
+*Child item selected, flyout open*
+
+![Child item selected, flyout open](images/commandbar2.PNG)
+
+*Child item selected, flyout closed*
+
+![Child item selected, flyout open](images/commandbar3.PNG)
+
+*Other top-level item selected*
+
+![Top-level item selected](images/commandbar1.PNG)	 
+
+### Feature #3. AlwaysExpanded API
+This will be a new API added to CommandBarFlyout that will give the developer the ability to keep the CommandBarFlyout in its expanded mode at all times.
+
+When the AlwaysExpanded property is set to true, the [...] button will not appear, and the user will not be able to collapse the CommandBarFlyout. Once the [...] button is collapsed, other AppBarButtons/CommandBarFlyout items should be able to take its space. 
+
+This property will only have an effect if the CommandBarFlyout has secondary commands. If there are no secondary commands, the CommandBarFlyout will always be in collapsed mode (hence why there’s no AlwaysCollapsed property). 
+
+CommandBarFlyout can still be collapsed/expanded by the developer programmatically even when this property is set to true. 
+
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+### Display a SplitButton in a CommandBarFlyout that opens its menu flyout to the right (using new style)
```

Comment:
There's no new API here, just new behavior for the SplitButton visualization?

Is there any way to know what the list of CB-friendly types are, other than reading the documentation?

Reply:
Correct, no new API. Just showing how to use the new (to-be-built) SplitButtonCommandBarFlyoutStyle. 

As for CB-friendly types, other than in the docs you can also find AppBarButton, AppBarToggleButton, and AppBarSeparator come up as examples in Intellisense when building a <CommandBar>.